USAGE.md

```diff
@@ -1,370 +1,707 @@
 
-# Mapping Concepts
-First, we must describe how ATT&CK objects and properties map to STIX 2.0 objects and properties.
+- [The ATT&CK Data Model](#The-ATT&CK-Data-Model), which describes the format of the data and highlights how it departs from the stock STIX2.0 format
+- [Accessing ATT&CK Data in Python](#Accessing-ATT&CK-Data-in-Python), which describes different methodologies that can be used to load the ATT&CK data into a script
+- [Python Recipes](#Python-Recipes), which provides python3 examples of common ways to query the ATT&CK data once loaded
 
-## Objects
-In ATT&CK, there are three main concepts (excluding Tactics for now): Techniques, Groups, and Software. Most techniques also have Mitigations. STIX 2.0 describes these as objects and uses different terminology to describe them. The following table is a mapping of ATT&CK concepts to STIX 2.0 objects:
+This document does not describe the generic STIX format, instead detailing how said format applies to ATT&CK and where ATT&CK departs from that format. To find out more about STIX, please see [the STIX 2.0 website](https://oasis-open.github.io/cti-documentation/stix/intro). 
 
-ATT&CK concept | STIX Object type
----------------|-----------------
-Technique | `attack-pattern`
-Group | `intrusion-set`
-Software | `malware` or `tool`
-Mitigation | `course-of-action`
-Tactic | `x-mitre-tactic`
-Matrix | `x-mitre-matrix`
+We also recommend reading the [ATT&CK Design and Philosophy Paper](https://attack.mitre.org/docs/ATTACK_Design_and_Philosophy_March_2020.pdf), which describes high-level overall approach, intention, and usage of ATT&CK.
 
-The above STIX types are found as literal strings assigned to the `type` property of the STIX JSON object. As shown in the table, in STIX 2.0, there are objects called "Course(s) of Action" used to describe mitigations to ATT&CK techniques. Similarly, the STIX 2.0 object called "Attack Pattern" describes techniques, etc. It should also be noted that Tactics are not an explicit object type in STIX 2.0, and they are referenced implicitly as kill chain phases within the other object types, as described in the tables below.
+# The ATT&CK Data Model
 
-## Properties
-The following is a table mapping of ATT&CK properties, the old ATT&CK MediaWiki names, and the new STIX properties. Some of these properties are standard STIX properties, while others were custom-created for compatibility with ATT&CK. These properties are accessed from STIX objects as JSON properties.
+The data in this repository is STIX2.0 and divided into three folders, one for each domain of ATT&CK. These domains generally follow the same format with a few departures. Domain differences will be noted in the relevant sections of this document. 
 
-### Migrating from MediaWiki
+STIX uses a different terminology than ATT&CK. The following table is a mapping of ATT&CK concepts to STIX 2.0 objects:
```

Comment:
Saying that ATT&CK uses a different terminology might be confusing, since ATT&CK actually uses the STIX terminology. How about:

ATT&CK uses a mix of predefined and custom STIX objects to implement ATT&CK concepts. The following table...

Reply:
The case I was trying to make here was that ATT&CK concepts have different words than the STIX types, e.g we say technique where in STIX they're attack-patterns. I can see how it is confusing however, and have adopted your proposed rewrite.